docker-hub/publish/index.md

```diff
@@ -2,28 +2,81 @@
+- Verified publisher badge
+- Insights and analytics
+- Advanced image management
+- Vulnerability scanning
+- No rate limiting
 
-The **Verified Publisher** badge included on the Docker Hub repositories indicates that these repositories are published by Docker partners that are part of the Verified Publisher program, and are qualified to be included in the developer secure supply chain. You can limit your repository search, when you use the 'Verified Publisher' filter.
+## Verified publisher badge
 
-To find out more about the Docker Verified and Publisher Program and to sign up for a partnership with Docker, visit the [Docker Verified Publisher Program](https://www.docker.com/partners/programs){:target="_blank" rel="noopener" class="_"} page.
+![Docker, Inc. org with a verified publisher badge](./images/verified-publisher-badge.png)
 
-![Verification Badge](images/VerifiedPublisherBadge.png)
+The verified publisher badge signals high quality, and trust, to developers.
+Images with this badge is verified to be of high quality, and that the content
+can be trusted.
+
+## Insights and analytic
+
+The [insights and analytics](/docker-hub/publish/insights-analytics){:
+target="blank" rel="noopener" class=""} service provides usage metrics for how
+the community uses your Docker images, and grants you insight into your user's
+behavior.
+
+![The insights and analytics tab on the Docker Hub website](./images/insights-and-analytics-tab.png)
+
+Select the time span for which you want to view analytics data, and export the
+data in either a summary or raw format. The summary format shows you image pulls
+per tag, and the raw format lists information about every image pull for the
+selected time span. Data points include tag, type of pull, user geolocation,
+client tool (user agent), and more.
+
+## Advanced image management
+
+[Advanced image management](/docker-hub/image-management/){: target="blank"
+rel="noopener" class=""} is a tool that lets publishers browse and manage their
+Docker Hub inventory. List, filter, and delete image tags in your repositories,
+using the dashboard or API.
+
+## Vulnerability scanning
+
+Automatic vulnerability scanning using Snyk for images published to Docker Hub.
+Scanning images ensures that the published content is secure, and underlines to
+developers that it can be trusted. Scanning can be enabled on a per-repository
+basis, refer to [vulnerability scanning](/docker-hub/vulnerability-scanning/){:
+target="blank" rel="noopener" class=""} for more information about how to use
+it.
+
+## No rate limiting
+
+For regular Docker images, image downloads are
+[rate limited](/docker-hub/download-rate-limit/){: target="blank" rel="noopener"
+class=""}. Images by verified publishers, however, are exempted from rate
+limiting; users can pull these images an unlimited number of times.
```

Comment:
**issue:** I believe this depends on the tier of DVP. Not **all** DVPs get exempted from rate limiting, maybe we should be more specific about it here?

Reply:
I added a clarification that rate limit exempt depends on the purchasing plan.